operator/pkg/util/cert/util.go

```diff
@@ -0,0 +1,246 @@
+// Copyright 2020 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package cert
+
+import (
```

Comment:
Is this  a brand new file ? Or derived from the security/ package ? If later, please add a comment, that code was already reviewed and tested.

Reply:
add comments for the copied method